packages/pnpm/test/recursive/outdated.ts

```diff
@@ -51,15 +51,16 @@ test('pnpm recursive outdated', async (t: tape.Test) => {
     t.equal(result.status, 0)
 
     t.equal(normalizeNewline(result.stdout.toString()), stripIndent`
-    ┌───────────────────┬─────────┬────────┬──────────────────────┐
-    │ Package           │ Current │ Latest │ Dependents           │
-    ├───────────────────┼─────────┼────────┼──────────────────────┤
-    │ is-negative       │ 1.0.0   │ 2.1.0  │ project-2            │
-    ├───────────────────┼─────────┼────────┼──────────────────────┤
-    │ is-negative (dev) │ 1.0.0   │ 2.1.0  │ project-3            │
-    ├───────────────────┼─────────┼────────┼──────────────────────┤
-    │ is-positive       │ 1.0.0   │ 3.1.0  │ project-1, project-3 │
-    └───────────────────┴─────────┴────────┴──────────────────────┘
+    ┌───────────────────┬─────────┬────────┬────────────┐
+    │ Package           │ Current │ Latest │ Dependents │
+    ├───────────────────┼─────────┼────────┼────────────┤
+    │ is-negative       │ 1.0.0   │ 2.1.0  │ project-2  │
+    ├───────────────────┼─────────┼────────┼────────────┤
+    │ is-negative (dev) │ 1.0.0   │ 2.1.0  │ project-3  │
+    ├───────────────────┼─────────┼────────┼────────────┤
+    │ is-positive       │ 1.0.0   │ 3.1.0  │ project-1  │
+    │                   │         │        │ project-3  │
```

Comment:
I am not sure how I feel about this change. Maybe it should go through a separate PR